client/profile/user-profile-overlay.tsx

```diff
@@ -40,70 +35,15 @@ export function ConnectedUserProfileOverlay({
   userId,
   popoverProps,
 }: ConnectedUserProfileOverlayProps) {
-  const dispatch = useAppDispatch()
-  const selfUser = useAppSelector(s => s.auth.user)
   const user = useAppSelector(s => s.users.byId.get(userId))
-  const party = useAppSelector(s => s.party)
-  const onPopoverDismiss = popoverProps.onDismiss
-
-  const onViewProfileClick = useCallback(() => {
-    navigateToUserProfile(user!.id, user!.name)
-  }, [user])
-
-  const onWhisperClick = useCallback(() => {
-    navigateToWhisper(user!.name)
-  }, [user])
-
-  const onInviteToPartyClick = useCallback(() => {
-    dispatch(inviteToParty(user!.id))
-    onPopoverDismiss()
-  }, [user, dispatch, onPopoverDismiss])
-
-  const onRemovePartyInvite = useCallback(() => {
-    dispatch(removePartyInvite(party.id, user!.id))
-    onPopoverDismiss()
-  }, [party, user, dispatch, onPopoverDismiss])
-
-  const onKickPlayerClick = useCallback(
-    userId => {
-      dispatch(kickPlayer(party.id, user!.id))
-      onPopoverDismiss()
-    },
-    [party, user, dispatch, onPopoverDismiss],
-  )
 
   if (!user) {
     return null
   }
 
-  const actions: React.ReactNode[] = []
-  actions.push(<MenuItem key='profile' text='View profile' onClick={onViewProfileClick} />)
-
-  if (user.id !== selfUser.id) {
-    actions.push(<MenuItem key='whisper' text='Whisper' onClick={onWhisperClick} />)
-
-    if (PARTIES && IS_ELECTRON) {
-      const isAlreadyInParty = party.members.has(user.id)
-      const hasInvite = party.invites.has(user.id)
-      if (isAlreadyInParty) {
-        actions.push(
-          <MenuItem key='kick-party' text='Kick from party' onClick={onKickPlayerClick} />,
-        )
-      } else if (hasInvite) {
-        actions.push(
-          <MenuItem key='invite' text='Uninvite from party' onClick={onRemovePartyInvite} />,
-        )
-      } else {
-        actions.push(
-          <MenuItem key='invite' text='Invite to party' onClick={onInviteToPartyClick} />,
-        )
-      }
-    }
-  }
-
   return (
     <UserProfileOverlay user={user} popoverProps={popoverProps}>
-      {actions}
+      {null}
```

Comment:
I don't think actions make sense ever again as of this PR, so you should probably just change the expected props of this component. Are we sharing this with the bottom left thing or something? If so they should probably be split, they aren't the same